### PR TITLE
Update rpc requests according to ckb 0.4.0

### PR DESCRIPTION
**Pull request by duanyytop**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,4 +4,10 @@
 
 Java SDK for Nervos [CKB](https://github.com/nervosnetwork/ckb).
 
+You can generate the jar and import manually.
+```
+git clone https://github.com/nervosnetwork/ckb-sdk-java.git
+gradle shadowJar
+```
+
 Note: this is at very early stage, treat it as a PoC implementation.
```
Comment (ashchan) [inline README.md]: I guess we can remove this line now that SDKs will keep update with CKB core.

File: core/src/main/java/org/nervos/ckb/exceptions/APIErrorException.java [hunk 2]
```diff
@@ -0,0 +1,16 @@
+package org.nervos.ckb.exceptions;
+
+import java.io.IOException;
+
+/**
+ * Created by duanyytop on 2018-12-26.
+ * <p>
```
Comment (ashchan) [inline core/src/main/java/org/nervos/ckb/exceptions/APIErrorException.java]: Is this `<p>` IDE specific? It looks like an open HTML element?

File: README.md (hunk 1 above)
Comment (duanyytop) [reply README.md]: OK, I remove it

File: core/src/main/java/org/nervos/ckb/exceptions/APIErrorException.java (hunk 2 above)
Comment (duanyytop) [reply core/src/main/java/org/nervos/ckb/exceptions/APIErrorException.java]: em..., IDE add automatically. I remove it

**Pull request merged**